formats/protobuf/commonMain/src/kotlinx/serialization/protobuf/schema/Generation.kt

```diff
@@ -0,0 +1,413 @@
+@file:OptIn(ExperimentalSerializationApi::class)
+
+package kotlinx.serialization.protobuf.schema
+
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.descriptors.*
+import kotlinx.serialization.protobuf.ProtoIntegerType
+import kotlinx.serialization.protobuf.ProtoNumber
+import kotlinx.serialization.protobuf.ProtoType
+import kotlin.native.concurrent.SharedImmutable
+
+/**
+ * Generate protocol buffers version 2 schema text for given serializable descriptors.
+ * All descriptors will be placed in one schema which can be saved in one `.proto` file.
+ * @param descriptors descriptors of serializable classes for which the scheme will be generated
+ * @param packageName protobuf common package for schema
+ * @param options protobuf options to include in schema
+ */
+@ExperimentalSerializationApi
+public fun generateProto2Schema(
```

Comment:
I double this, let's stick to 'generateProtobufSchema'

Reply:
I don't like 'Schema' very much because one may think that this method returns a meaningful representation of the schema in Kotlin classes (like JsonElement), but it returns String. I'd add 'Text' or something else as a suffix to highlight that result of this method is useful for only one purpose — to write it to `.proto` file